tests/resources/Util.robot

```diff
@@ -75,7 +76,7 @@ Install VIC Appliance To Test Server
 
     # Install the VCH now
     Log To Console  \nInstalling VCH to test server...
-    ${output}=  Run VIC Machine Command  ${certs}  ${vol}  network  'VM Network'
+    ${output}=  Run VIC Machine Command  ${certs}  ${vol}  ${bridge}  ${external}
```

Comment:
Maybe what I've done now is what you were suggesting but in a different way - it makes sure the env vars are set regardless. More ugly, but less error prone